src/locales/rw/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,627 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+msgid ""
+msgstr ""
+"Project-Id-Version: PROJECT VERSION\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2020-08-07 15:22+0200\n"
+"PO-Revision-Date: 2020-08-07 12:17+0200\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language: rw\n"
+"Language-Team: rw <LL@li.org>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.8.0\n"
+
+#: services/messenger.py:40
+msgid "Friend"
+msgstr "Nshuti"
+
+#: services/messenger.py:75
+#, python-format
+msgid ""
+"🙏🏼 Hi %(first_name)s, so you’ve decided to make your first steps in Open "
+"Source. That’s great."
+msgstr ""
+"🙏🏼 Muraho% (first_name), nuko rero wahisemo gutera intambwe zawe za mbere"
+" muri OpenSource. Nibyiza."
+
+#: services/messenger.py:86 services/messenger.py:201 services/messenger.py:221
+msgid "✔️ Yes"
+msgstr "✔️ Ego"
+
+#: services/messenger.py:87 services/messenger.py:203 services/messenger.py:222
+msgid "❌ Not yet"
+msgstr "❌ Oya"
+
+#: services/messenger.py:90
+#, python-format
+msgid "So, tell me %(first_name)s do you know what Open Source is? 👇🏼"
+msgstr "Noneho, mbwira% (first_name) uzi Open source icyo aricyo? 👇🏼"
+
+#: services/messenger.py:102
+msgid "Open Source 🔓"
+msgstr ""
+
+#: services/messenger.py:104
+msgid "Git"
+msgstr ""
+
+#: services/messenger.py:105 services/messenger.py:808
+msgid "GitHub"
+msgstr ""
+
+#: services/messenger.py:106
+msgid "Make a PR"
+msgstr "Gukora PR"
+
+#: services/messenger.py:108
+msgid "FB Open Source"
+msgstr ""
+
+#: services/messenger.py:110
+msgid "Fork me on GitHub"
+msgstr "Kora fork kuri GitHub"
+
+#: services/messenger.py:146
+msgid ""
+"Oh you need some help 🆘! This is the main menu, select what you need "
+"below 👇🏼"
+msgstr ""
+"Yoo ukeneye ubufasha 🆘! Iyi ni menu nyamukuru, hitamo icyo ukeneye.munsi "
+"👇🏼"
+
+#: services/messenger.py:155
+#, python-format
+msgid ""
+"I didn't get you %(first_name)s!\n"
+"You said : %(msg)s\n"
+"\n"
+" This is the main menu, select what you need below 👇🏼"
+msgstr ""
+"Sinumvise %(first_name)s!\n"
+"Wavuze:% (msg)s\n"
```

Comment:
@Emile-Nsengimana error `... %(msg)s\n`

Reply:
I saw you have already fix the bug @elinguiuriel 🙏